ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -20,7 +22,7 @@ class PPOTrainer(RLTrainer):
 
     def __init__(
         self,
-        brain,
+        brain_name,
```

Comment:
Ditto for type annotations

Reply:
it's odd that the trainers have escaped type annotations for this long.